client/src/components/Donation/patreon-button.tsx

```diff
@@ -30,6 +36,7 @@ const PatreonButton = (): JSX.Element | null => {
       className='patreon-button link-button'
       data-patreon-widget-type='become-patron-button'
       href={href}
+      onClick={postPatreonRedirect}
```

Comment:
@ahmadabdolsaheb If you are sure about giving an `a` element an `onclick`, then I am fine with it, but I would advise this be a `button` with a side-effect redirecting to `href`.

Reply:
Isn't that an a11y no-no, though, making an `a` into a `button`?  This really is just a link that's dressed up to look like a button.

Reply:
> Isn't that an a11y no-no, though, making an a into a button?

Yes, but it goes both ways. This is not just a hyperlink for navigation, it invokes a `postPatreonRedirect` side-effect.

Reply:
True enough.  It's kind of a weird hybrid, but it seems to be primarily a link (I click on it and my browser heads of to Patreon).

I don't know enough a11y stuff to be sure, though.

Reply:
I think it should be fine.